examples/sdk/train_api.py

```diff
@@ -0,0 +1,68 @@
+from kubeflow.training.api.training_client import TrainingClient
+from kubeflow.storage_initializer.hugging_face import (
+    HuggingFaceModelParams,
+    HuggingFaceTrainParams,
+    HfDatasetParams,
+)
+from peft import LoraConfig
+import transformers
+from transformers import TrainingArguments
+
+client = TrainingClient(
+    config_file="/Users/deepanker/Downloads/deepanker-test-kubectl.cfg"
```

Comment:
```suggestion
    config_file="/Users/deepanker/Downloads/deepanker-test-kubectl.cfg"
```

This wouldn't work fine.